test/sql/aggregate/group/test_group_by_alias.test

```diff
@@ -65,11 +65,15 @@ NULL	NULL	NULL
 2	0	2.000000
 3	1	3.000000
 
-# cannot use GROUP BY column in an aggregation...
-statement error
-SELECT i % 2 AS k, SUM(k) FROM integers GROUP BY k
+# allow GROUP BY derived column in an aggregation
```

Comment:
This now works fine. Is there any reason to prevent this? Special-casing it to fail seems fiddly.

Reply:
That should indeed work now